### PR TITLE
Added tox configuration to check tests coverage

### PR DESCRIPTION
**Pull request by armstrongmsg**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -12,6 +12,13 @@ commands = py.test
 deps =
     -r{toxinidir}/requirements.txt
 
+[testenv:coverage]
+basepython = python3
```
Comment (javanlacerda) [inline tox.ini]: Change the python version to 3.7, please.

File: tox.ini [hunk 2]
```diff
@@ -12,6 +12,13 @@ commands = py.test
 deps =
     -r{toxinidir}/requirements.txt
 
+[testenv:coverage]
```
Comment (javanlacerda) [inline tox.ini]: Add "coverage" to envlist

**Review by javanlacerda — CHANGES_REQUESTED** [review]
Can you add the step "Coverage" to the Jenkins pipeline? You can do it adding to the Jenkinsfile. 
P.S: You can use the command "tox -e coverage". Thanks!

File: tox.ini (hunk 1 above)
Comment (armstrongmsg) [reply tox.ini]: Done

File: tox.ini (hunk 2 above)
Comment (armstrongmsg) [reply tox.ini]: Done

**Review by javanlacerda — APPROVED** [review]
LGTM!

**Review by DiegoGamaV — APPROVED** [review]

**Pull request merged**